argoverse/data_loading/object_label_record.py

```diff
@@ -207,6 +238,56 @@ def draw_rect(selected_corners: np.array, color: Tuple[int, int, int]) -> None:
         return img
 
 
+def add_text_cv2(img: np.ndarray, text: str, x: int, y: int, color: Tuple[int,int,int], thickness: int = 3) -> None:
+    """Add text to image using OpenCV. Color should be RGB order"""
+    img = cv2.putText(
+        img,
+        text,
+        (x,y),
+        fontFace=cv2.FONT_HERSHEY_SIMPLEX,
+        fontScale=1,
+        thickness=thickness,
+        color=color
+    )
+
+
+def draw_alpha_rectangle(
+    img: np.ndarray,
+    top_left: Tuple[int,int],
+    bottom_right: Tuple[int,int],
+    color_rgb: Tuple[int,int,int],
+    alpha: float
+):
+    """Alpha blend colored rectangle into image. Coords given as (x,y) tuples"""
+    img_h, img_w, _ = img.shape
+    mask = np.zeros((img_h,img_w),dtype=np.uint8)
+    mask[top_left[1]:bottom_right[1],top_left[0]:bottom_right[0]] = 1
+    return vis_mask(img, mask, np.array(list(color_rgb[::-1])), alpha)
+
+
+def vis_mask(img: np.ndarray, mask: np.ndarray, col: Tuple[int,int,int], alpha: float=0.4):
```

Comment:
Was `black` run on this file? Some of the spacing looks a bit different than their styling.

Reply:
not yet, i'll run it now